### DISCUSSION TITLE
Overriding CMS view

### DISCUSSION THREAD

**Discussion by jsebaf** [wintercms/winter#1136] [Q&A] [answered]
Hello!

I need to disable or hide the "Reset Defaults" button in the themeoptions CMS controller. I'm wondering what is the correct way to do this without directly modifying the CMS code.

Thanks

**Comment by bennothommo** [comment, marked as answer]
Probably the easiest way to do this is to create a Backend skin and overwrite the Theme options page template. It's not currently documented in the Winter docs, but you can look at the [Tailwind UI plugin](https://github.com/wintercms/wn-tailwindui-plugin) for inspiration on how to create your own skin.

The dot points are:
- A skin is defined as part of a plugin, usually in the `skins` folder.
- You will need to instruct your plugin to add a [view path](https://github.com/wintercms/wn-tailwindui-plugin/blob/main/Plugin.php#L117) for modules and plugins in your skin folder, so Winter will look in your skin assets for a template first, before falling back to the standard template.
- You would then need to overwrite the `modules/cms/controllers/themeoptions/update.php` template with your own copy within your skin, removing (or disabling) the reset button in your copy.
- You can either [load the skin automatically](https://github.com/wintercms/wn-tailwindui-plugin/blob/main/Plugin.php#L98) like the Tailwind UI plugin does, or you can simply change the `backendSkin` configuration value in `config/cms.php` in your app to the fully namespaced class of your Skin class.

Hopefully that should be enough to go on for now, however, we will endeavour to document this at some point.

**Comment by jsebaf** [comment]
It works!
Thanks!

**Reply by bennothommo** [reply]
Great! Just so you aware, you will need to keep track of any changes to the standard template for this page going forward, and ensure that those changes are replicated in your copy, hence why we don't recommend overriding the standard templates in most cases. That being said - it's fairly unlikely we'll make changes to that page often.
